packages/spinup-cli/package.json

```diff
@@ -1,19 +1,14 @@
 {
-  "version": "0.0.1",
-  "name": "@gridsome/spinup-cli",
-  "description": "A command line tool for creating portals at Rebilly.",
-  "homepage": "https://github.com/Rebilly/rebilly-spinup/tree/master/packages/spinup-cli#readme",
-  "repository": "https://github.com/Rebilly/rebilly-spinup/tree/master/packages/spinup-cli",
+  "name": "@rebilly/spinup-cli",
+  "version": "0.1.2",
```

Comment:
why the jump to `0.1.2`, shouldn't it be `0.1.0`?

Reply:
I was testing to make sure the global install worked and picked up the version number between updates.